### PR TITLE
add rename script

### PR DESCRIPTION
**Pull request by danfitz36**
this is used to rename all masks/paths to align with artboards and ensure the following scripts work as intended. it still needs a little cleanup

### CODE REVIEW

File: renamePaths.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+
+if (app.documents.length > 0) {
+  var bkDoc = app.activeDocument;
+
+  // check for breakmark template
+  if (bkDoc.name == "breakmark-template.ai") {
+    var allArtboards = bkDoc.artboards;
+    app.selection = null;
```
Comment (shamshirz) [inline renamePaths.js:8]: What is `app.selection`? we iterate on it below, but I don't see it getting set to anything?

File: renamePaths.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+    for (var i = 0; i < allArtboards.length; i++) {
+      currentName = allArtboards[i].name;
+      allArtboards.setActiveArtboardIndex(i);
+      bkDoc.selectObjectsOnActiveArtboard();
+
+      for (var j = 0; j < app.selection.length; j++) {
```
Comment (shamshirz) [inline renamePaths.js:15]: Nesting loops can get confusing really quickly, when I get a chance I'm going to pull part of this into a separate function

File: renamePaths.js (hunk 1 above)
Comment (danfitz36) [reply renamePaths.js:8]: `bkDoc.selectObjectsOnActiveArtboard();` is an illustrator function that adds objects to `app.selection`

File: renamePaths.js (hunk 2 above)
Comment (danfitz36) [reply renamePaths.js:15]: so we loop through each artboard, select all on the active one, do a thing on each of the selected objects, then move to the next artboard.

**Pull request merged**